docs/catalog/cloud/Use energy efficient hardware.md

```diff
@@ -0,0 +1,42 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:software-engineer
+- role:cloud-engineer
+- size:medium
+---
+
+# Use energy efficient hardware
```

Comment:
This is very generic as a title. Consider something more specific to this pattern like "Use cloud native processors VMs"